goal_src/engine/pc/pckernel.gc

```diff
@@ -135,6 +144,16 @@
   (update-from-os obj)
   (update-to-os obj)
 
+  (let ((info (new 'stack 'discord-info)))
+    (set! (-> info fuel) (&-> *game-info* fuel))
+    (set! (-> info money-total) (&-> *game-info* money-total))
+    (set! (-> info buzzer-total) (&-> *game-info* buzzer-total))
+    (set! (-> info status) "Playing Jak and Daxter: TPL")
+    (set! (-> info level) (symbol->string (-> (-> *target* current-level) name)))
```

Comment:
I think this could crash if there's no `*target*` spawned.  In this case, `*target*` will be `#f`  You could do something like:
```
(if *target*
    (symbol->string (-> *target* current-level name))
    "Unknown Level"
    )
```

if you want to also get the level that the camera is in, you could do:
```
(-> (level-get-target-inside *level*) name)
```
which, despite the name, the `level-get-target-inside` method will use the camera position if `*target*` isn't spawned.

Also, you can use `->` with multiple dereferences, so `(-> (-> a b) c)` is the same as `(-> a b c)`.

If you want to test with no target spawned, you can just run `(stop #f)` and it will kill the `*target*` process.